### PR TITLE
Mentoring program announcement

### PR DESCRIPTION
**Pull request by petrasovaa**
News item about launching NSF funded mentoring program.

### CODE REVIEW

**Review by wenzeslaus — APPROVED** [review]
For the wiki, I extracted the relevant NSF logo usage text. It might be good to include that into the commit message.

```
NSF Logos and Usage Standards

NSF's logo can be used by recipients of NSF support for the sole purpose of acknowledging that support.

NSF's logo can be used to link to an NSF website or acknowledge NSF assistance or affiliation.

NSF's logo cannot be used in a manner that falsely implies employment by or affiliation with NSF. NSF's logo cannot be used to imply or endorse a product or service.

No explicit permission is necessary to use the NSF logo.

The NSF logo is the sole visual identifier of the U.S. National Science Foundation and embodies a set of values that is applied to everything we create and communicate.

Source: https://www.nsf.gov/policies/logos.jsp (2023-10-09) 
```

File: content/news/2023_10_11_Mentoring_Program_announced.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+The use cases include (but are not limited to):
+
+* developing workflow that uses GRASS GIS e.g., through Python, Bash, R API,
+* contributing a research tool for GRASS GIS in Python, C, or C++,
+* running GRASS GIS on a high-performance cluster,
+* integrating GRASS GIS in another software.
+
+
+The program is open to people from all walks of life regardless of employment or student status. Projects on beginner technical level are welcome.
```
Comment (veroandreo) [inline content/news/2023_10_11_Mentoring_Program_announced.md:25]:
```suggestion
The program is open to people from all walks of life regardless of employment or student status. Applicants and projects on beginner technical level are welcome.
```
Would that make sense?

File: content/news/2023_10_11_Mentoring_Program_announced.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Interested? To apply, follow the directions on the [Mentoring Program wiki](https://grasswiki.osgeo.org/wiki/Mentoring_Program).
+
+<a href="/images/logos/NSF_logo.png">
+  <img src="/images/logos/NSF_logo.png"
+   alt="U.S. National Science Foundation logo"
+   title="Supported by U.S. National Science Foundation"
+   width="15%" style="float:right;padding-left:25px;padding-top:15px">
+</a>
+
+The funding to cover the time of mentors is provided by the U.S. National Science Foundation, award 2303651.
```
Comment (veroandreo) [inline content/news/2023_10_11_Mentoring_Program_announced.md:36]:
```suggestion
The funding to cover the mentors' time is provided by the U.S. National Science Foundation, award 2303651.
```

**Review by veroandreo — APPROVED** [review]
I left minor suggestions :)

**Comment by neteler** [comment]
Now online:
https://grass.osgeo.org/news/2023_10_11_mentoring_program_announced

**Pull request merged**